Sources/ClientRuntime/Networking/Http/HttpContent.swift

```diff
@@ -0,0 +1,33 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import AwsCommonRuntimeKit
+
+/// A wrapper around a StreamReader that conforms to the IStreamable protocol
+/// This is used to pass the StreamReader to the CRT HttpMessage
+public class HttpContent: IStreamable {
+    internal let stream: StreamReader
```

Comment:
We haven't been explicitly defining methods as `internal` since `internal` is the default. We should either remove `internal` here or file an issue to update all internal APIs to explicitly define `internal`.

Reply:
for now removing internal but would like to discuss what team think on adding it explicitly.

Reply:
Definitely not opposed to it! Just want consistency